### DISCUSSION TITLE
Feature request: Add preset shadows, blur, grids and layouts to add to new shapes

### DISCUSSION THREAD

**Discussion by Manueljlin** [uxbox/uxbox#524] [Ideas]
This feature is a bit more complex but it would save a lot of time. Basically, it would be a way to store shadows, blur, grids and layouts on a list to choose and apply to different shapes, paths or artboards and apply them on one click. Then being able to update the master one and updating auttomatically on all shapes. Figma handles it this way:

1- There's a shape in the canvas with no effect
![image](https://user-images.githubusercontent.com/11275573/106601124-9586c980-655b-11eb-82e1-1c4620acbdda.png)

2- You add a shadow, blur or other effect
![image](https://user-images.githubusercontent.com/11275573/106601226-b5b68880-655b-11eb-858d-b9984cd63685.png)
 
3- Then add it to a master preset list
![image](https://user-images.githubusercontent.com/11275573/106601417-e4ccfa00-655b-11eb-96e6-c7adf5664e97.png)

4- Add another shape
![image](https://user-images.githubusercontent.com/11275573/106601537-0ded8a80-655c-11eb-8b74-4f89cd0aa8cf.png)

5- Apply the same shadow, blur, etc
![image](https://user-images.githubusercontent.com/11275573/106601655-31b0d080-655c-11eb-9ca2-6045d7ba33ae.png)
![image](https://user-images.githubusercontent.com/11275573/106601737-4db47200-655c-11eb-8cb4-90c74efc8310.png)

6- Modify the master effect and update on all objects at the same time
![image](https://user-images.githubusercontent.com/11275573/106601866-750b3f00-655c-11eb-9038-4e764a513667.png)

Thanks a lot for this awesome app, it's super powerful for being an alpha release <3

**Comment by elhombretecla** [comment]
Hi again Manuel!!! You are a PRO user!
We already have an User Story for this :)
Effects will become part of the libraries and assets panel soon.

**Reply by Manueljlin** [reply]
Cool, thanks a lot :D
